sklearn/decomposition/factor_analysis.py

```diff
@@ -88,6 +88,10 @@ class FactorAnalysis(BaseEstimator, TransformerMixin):
         Number of iterations for the power method. 3 by default. Only used
         if ``svd_method`` equals 'randomized'
 
+    rotation : None | 'varimax' | 'orthomax'
+        If not None, apply the indicated rotation. Currently, varimax and
+        orthomax are implemented.
```

Comment:
Should there be a reference or some discussion in the user guide? At least there should be a link to the example from the user guide